chart/BUILD.yaml

```diff
@@ -14,9 +14,9 @@ packages:
       - imageRepoBase
     config:
       commands:
-        # set values
-        - ["yq", "w", "-i", "Chart.yaml", "version", "${version}"]
-        - ["yq", "w", "-i", "values.yaml", "version", "${version}"]
+        # set values. Use the value in values.yaml:.version as ${version}
+        - ["sh", "-c", "yq w -i Chart.yaml version 0.1.0-${version}"]
+        - ["sh", "-c", "yq w -i values.yaml version 0.1.0-${version}"]
```

Comment:
This seems not to work. When we set the version in the `values.yaml` like this, it tries to fetch images like that:
```
eu.gcr.io/gitpod-core-dev/build/messagebus:0.1.0-gpl-bump-helm.14
```
But these tags do not exist. It's just:
```
eu.gcr.io/gitpod-core-dev/build/messagebus:gpl-bump-helm.14
```

Isn't it sufficient if we change the version in the `Chart.yaml` file to `0.1.0-${version}` and leave the version in the `values.yaml` file to `${version}`? That should work, shouldn't it?

Reply:
Also a nit: The comment

> Use the value in values.yaml:.version as ${version}

is outdated.